timetracker/vms/views.py

```diff
@@ -79,6 +79,13 @@ class DashboardView(LoginRequiredMixin, TemplateView):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
+        # get all admin agencies associtated with that user.
+        try:
+            adminagency = models.StaffingAgencyAdmin.objects.filter(
+                user=self.request.user)
+            context['staff_admin'] = adminagency
+        except adminagency.DoesNotExist:
```

Comment:
Alright so I did not do it that way out of the off chance there was an client admin with more than one instance for each company.  there shouldn't be but it doesn't quite break if it does.  I realize that I could prob use `exist()` instead to check if there was one that exist

Reply:
The way you have it right now, `adminagency` (and `context['staff_admin']`) is a queryset of staffing agencies, not a single instance.

Reply:
yeah it's a queryset of staffing agency admin.  I did not mean more than one admin per agency.  I meant if there is more than one entry for a single username.  IE.  is a admin for company1&company2 for whatever reason.  I can just use get but I was worried if there was a case where it would break.